613 welcome content

IanMayo opened the pull request:
Supports #613 

Note: this PR introduces a parsing problem.  Our `sanitize_filename` logic fails when trying to sanitize a file that sits in the root folder.

IanMayo committed:
introduce links to welcome content, so the DITA gets generated

IanMayo committed:
welcome is no longer DITA, it's now HTML. So, we don't need to copy it over

IanMayo committed:
introduce new welcome content

robintw committed:
Fix error in sanitise_filename where it failed on a '.' (ie current directory) input. Also fix a couple of links in the new Welcome.html.

robintw commented:
I've added a commit here to fix the sanitise_filename issue, and also at the same time I fixed a couple of relative links in Welcome.html that didn't work now it had been moved.

IanMayo commented:
> I've added a commit here to fix the sanitise_filename issue, and also at the same time I fixed a couple of relative links in Welcome.html that didn't work now it had been moved.

That's perfect - thanks @robintw
